{"name": "Тест на получение ошибки при попытке создании двух одинаковых пользователей", "status": "passed", "description": "Создаем последовательно двух пользователей передавая одинаковые данные емайл, пароль, имя и проверяем что возвращается ошибка", "start": 1747223382646, "stop": 1747223383264, "uuid": "451931fe-06c9-498c-a1dd-d629159a2fd3", "historyId": "80bf8b7c52b4eeac1ab702f4cf740319", "testCaseId": "80bf8b7c52b4eeac1ab702f4cf740319", "fullName": "tests.test_user.TestCreateUser#test_create_duplicate_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "LAPTOP-3QFAQEQR"}, {"name": "thread", "value": "6276-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}